### PR TITLE
Adding a rake task to export workspace data filtering by status.

### PR DESCRIPTION
**Pull request by caiosba**
## Description

This is an internal request, so, more details are in the ticket CV2-3600.

## How has this been tested?

I executed the rake task locally for a couple of workspaces and the CSVs were generated as expected.

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have added unit and feature tests, if the PR implements a new feature or otherwise would benefit from additional testing
- [ ] I have added regression tests, if the PR fixes a bug
- [x] I have added logging, exception reporting, and custom tracing with any additional information required for debugging
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] If I added a third party module, I included a rationale for doing so and followed our current [guidelines](https://meedan.atlassian.net/wiki/spaces/ENG/overview#Choose-the-%E2%80%9Cright%E2%80%9D-3rd-party-module)

### CODE REVIEW

File: lib/tasks/data/export_workspace_data.rake [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        else
+          return false
+        end
+      rescue StandardError => e
+        puts "Error uploading S3 object: #{e.message}"
+        return false
+      end
+
+      slugs = params.to_a
+      slugs.each do |slug|
```
Comment (melsawy) [inline lib/tasks/data/export_workspace_data.rake:37]: I suggest to query teams here i.e. Team.where(slug: slugs).find_each do |team|

File: lib/tasks/data/export_workspace_data.rake [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          puts "[#{i}/#{n}] Exporting item from workspace #{slug}..."
+          status = pm.status_i18n
+          if statuses_to_export.include?(status)
+            row = [
+              pm.claim_description&.description,
+              status,
+              pm.author_name,
+              pm.created_at.strftime("%Y-%m-%d %H:%M:%S"),
+              pm.published_at&.strftime("%Y-%m-%d %H:%M:%S"),
+              pm.linked_items_count(true),
```
Comment (melsawy) [inline lib/tasks/data/export_workspace_data.rake:62]: why you need to recalculate cached field? it'll be faster to call cached field without true

**Review by melsawy — CHANGES_REQUESTED** [review]

File: lib/tasks/data/export_workspace_data.rake (hunk 1 above)
Comment (caiosba) [reply lib/tasks/data/export_workspace_data.rake]: Thanks, changed!

File: lib/tasks/data/export_workspace_data.rake (hunk 2 above)
Comment (caiosba) [reply lib/tasks/data/export_workspace_data.rake]: Ah yes, I did it for testing... fixed now, good catch.

**Comment by caiosba** [comment]
Thanks @melsawy - I addressed the comments, please re-review.

**Review by melsawy — APPROVED** [review]

**Pull request merged**